src/Compilers/CSharp/Portable/BoundTree/BoundExpressionWithNullability.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal sealed partial class BoundExpressionWithNullability : BoundExpression
+    {
+        public override bool IsSuppressed
+        {
+            get => Expression.IsSuppressed;
+            protected set => throw ExceptionUtilities.Unreachable;
```

Comment:
Should we mark the setter as `[Obsolete]`?

Reply:
Hmm I'm not sure that would really work here. 

---
In reply to: [252448228](https://github.com/dotnet/roslyn/pull/32947#discussion_r252448228) [](ancestors = 252448228)